tox.ini

```
@@ -14,7 +14,7 @@ envlist =
 [testenv]
 deps = {[base]testdeps}
 extras = plotly
-commands = pytest tests
+commands = pytest {posargs}
```

```suggestion
commands = pytest tests {posargs}
```
Should we keep `tests`?

---

It would be needed if there was files with names like `test_*.py` in other directories than in `tests`. But it is not the case so it useless and removing it allows to run one specific test with command like `tox -e py38 -- tests/features/test_morphology.py` for example (if we set `tests` in the tox command, all the tests will always be run).

---

ok, lets keep as is